### PR TITLE
Ipv6 document

### PR DESCRIPTION
**Pull request by axzxc1236**
A little bit more polished version of https://github.com/qdm12/gluetun/issues/1648#issuecomment-1607331844

I agree to contribute in MIT license.

### CODE REVIEW

**Comment by axzxc1236** [comment]
English is not my first language, sorry if there is bad grammar in documentation and commit name.

File: setup/advanced/ipv6.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      "ipv6": true,
+      "fixed-cidr-v6": "fd00::/80",
+      "experimental": true,
+      "ip6tables": true
+    }
+    ```
+
+    [Reference: Docker documentation on IPv6](https://docs.docker.com/config/daemon/ipv6/#use-ipv6-for-the-default-bridge-network)
+
+    ⁉️ IP range change to unique local address following the note at button of the documentation.
```
Comment (qdm12) [inline setup/advanced/ipv6.md:20]: What do you mean by this?

File: setup/advanced/ipv6.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+1. Restart the Docker daemon to reload its JSON configuration. Most Linux distributions use `sudo systemctl restart docker` to do this.
+1. Edit your Gluetun `docker-compose.yml` and add the `sysctls` section and modify `WIREGUARD_ADDRESSES` to have both an IPv4 and an IPv6 address:
```
Comment (qdm12) [inline setup/advanced/ipv6.md:22]: > modify `WIREGUARD_ADDRESSES` to have both an IPv4 and an IPv6 address

Why is this needed? 🤔 
Would it work with only an IPv6 address?

Comment (qdm12) [inline setup/advanced/ipv6.md:22]: > and add the `sysctls` section

Are you sure we need both the modification to `/etc/docker/daemon.json` and the `sysctls` block? The comment you previously linked was mentioning to do either one or the other 🤔

File: setup/advanced/ipv6.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        sysctls:
+          - net.ipv6.conf.all.disable_ipv6=0
+    ```
+
+1. Test your setup:
+    1. Launch your docker-compose stack
+    1. Run:
+
+        ```sh
+        sudo docker run --rm --network=container:gluetun alpine:3.18 sh -c "apk add curl && curl -6 --silent https://ipv6.ipleak.net/json/"
```
Comment (qdm12) [inline setup/advanced/ipv6.md:41]: I changed this to directly curl ipleak over ipv6, this should work nicely (can you try it please 🙏 I don't have ipv6 over here, shame on me 😢

**Review by qdm12 — COMMENTED** [review]
Awesome thanks for the PR 💯 

I have pushed a few commits to fix the formatting and review the entire document (a bit of grammar, mostly simplifications/shorter document).

I still have a few questions, feel free to just answer them and I'll patch up the document one last time.

No worry about the commit messages, they all get squashed before merge anyway 😉

File: setup/advanced/ipv6.md (hunk 1 above)
Comment (axzxc1236) [reply setup/advanced/ipv6.md:20]: There is a section at the button of the page



So In my documentation and testing instead of using "2001:db8:1::/64" I used "fd00::/80"

File: setup/advanced/ipv6.md (hunk 2 above)
Comment (axzxc1236) [reply setup/advanced/ipv6.md:22]: Happens when you only give gluetun only IPv6 address, and I think the error message makes sense.

File: setup/advanced/ipv6.md (hunk 3 above)
Comment (axzxc1236) [reply setup/advanced/ipv6.md:41]: Container do change depending on the directory name of the directory of docker-compose.yml



When I put docker-compose.yml in "gluetun" directory, the name is "gluetun-gluetun-1".

When I put docker-compose.yml in "ipv6" directory, the name is "ipv6-gluetun-1".

File: setup/advanced/ipv6.md (hunk 2 above)
Comment (axzxc1236) [reply setup/advanced/ipv6.md:22]: The picture is what I see when I don't have sysctls section

File: setup/advanced/ipv6.md (hunk 3 above)
Comment (axzxc1236) [reply setup/advanced/ipv6.md:41]: I am not sure if the changing container name has anything to do with me using `sudo docker-compose up`

Comment (axzxc1236) [reply setup/advanced/ipv6.md:41]: I can make sure gluetun's container is "gluetun" by using `sudo docker-compose run --rm --name gluetun gluetun`

Comment (qdm12) [reply setup/advanced/ipv6.md:41]: Awesome, what matters is that curl command works well 💯

You can just set `name: gluetun` in your docker-compose.yml to always have the same name for the container. Otherwise yeah it's directory name + compose spec definition name + index.

File: setup/advanced/ipv6.md (hunk 2 above)
Comment (qdm12) [reply setup/advanced/ipv6.md:22]: Cool, leaving it as is then!

Comment (qdm12) [reply setup/advanced/ipv6.md:22]: Makes sense, IPv4 traffic doesn't make it through. I guess it would be interesting to have it working fully on IPv6, but another day perhaps!

File: setup/advanced/ipv6.md (hunk 1 above)
Comment (qdm12) [reply setup/advanced/ipv6.md]: Ok I reworded this, hopefully it will clear up things! 👍

**Review by qdm12 — APPROVED** [review]
